doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -1096,8 +1096,7 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
```

Comment:
Some things to do in the doc:

Now you should add a separate `.. cmd::` for "Scheme Equality".  The simplest option is just to put them together, e.g.

![image](https://user-images.githubusercontent.com/1253341/128279605-2ff0c6c6-9953-4e85-b680-2d4da1b968a2.png)

Second choice is to create a separate, rather short command entry that could refer back to "Scheme".  The current description for "Scheme" is a bit thin--there's no description or example of Elimination or Case.  If you can easily improve it, feel free to do so.  But it doesn't have to be in this PR.

Reply:
I'll take this as an opportunity to improve the documentation further.